lib/configurator.js

```diff
@@ -39,7 +39,7 @@ exports.DEFAULT_CONFIG = {
   host: 'in.treasuredata.com',
   logging: true,
   pathname: '/js/v3/event/',
-  protocol: document.location.protocol === 'https:' ? 'https:' : 'http:',
+  protocol: 'https:',
```

Comment:
Because the `DEFAULT_CONFIG` gets overridden, we should just force it to be `https` in all cases and not even respect `protocol` being passed in.